Scene/WorldSelector.tscn

```diff
@@ -22,7 +24,16 @@ layout_mode = 1
 visible = false
 layout_mode = 1
 
+[node name="SettingsSelector" parent="." instance=ExtResource("5_opwul")]
+visible = false
+layout_mode = 1
+
+[node name="AudioSelector" parent="." instance=ExtResource("6_5055g")]
+visible = false
+layout_mode = 1
```

Comment:
If I understand correctly, this `AudioSelector` scene is now unused, and should be removed from the scene, with `AudioSelector.tscn` and `audioSelector.gd` deleted.